apps/obsidian/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@discourse-graphs/obsidian",
```

Comment:
can't just use `obsidian` because the npm obsidian package is in our dependencies *sad face*.  will probably namespace all of the plugins this way